contents/blog/posthog-vs-mixpanel.md

```diff
@@ -0,0 +1,968 @@
+
+- [Who should self-host PostHog?](#who-should-self-host-posthog)
+- [How long does it take to deploy PostHog?](#how-long-does-it-take-to-deploy-posthog)
+- [Does PostHog user cookies?](#does-posthog-use-cookies)
+- [How much does PostHog cost?](#how-much-does-posthog-cost)
+- [How much does Mixpanel cost?](#how-much-does-mixpanel-cost)
+
+### Who should self-host PostHog?
+
+We recommend 95% of companies use [PostHog Cloud](/pricing) – it's fully-managed by us and you can choose [PostHog Cloud EU](/eu) if you need to keep data within the EU. The most common reasons for choosing to self-host PostHog are companies that handle very sensitive data, such as medical or defense information. 
+
+Enterprises looking to self-host will require an infrastructure team knowledgeable in Kubernetes and ClickHouse. See our [enterprise self-hosted docs](/docs/self-host/enterprise/overview) and [book a call](book-a-demo) to discuss your options and whether self-hosting is right for you. We also have a [partner directory](/partners) includes third-party specialists who can support self-hosted deployments.
+
+### How long does it take to deploy PostHog?
+
+When using PostHog Cloud, mere minutes. All you need to do is include a tracking snippet in your website and you're good to go. See our guide to [getting started on PostHog Cloud](/docs/getting-started/cloud) for more info.
+
+### Does PostHog use cookies?
+
+Yes, PostHog (like Mixpanel) uses cookies. Cookies are necessary to enable common use cases, such as cross-domain tracking and tracking returning visitors.
+
+It is possible, however, to configure PostHog to track users without requiring [without cookie banners](/tutorials/cookieless-tracking) by storing cookies in `memory` so it doesn't persist beyond an individual session.
+
+Both Mixpanel and PostHog also support anonymizing data, while the PostHog [Property Filter app](/apps/property-filter) allows you to limit the types of information collected to protect user privacy.
+
+### How much does PostHog cost?
+
+We believe in transparent pricing, which is why [our pricing page](/pricing) includes an easy-to-understand pricing calculator.
+
+PostHog Cloud includes a generous 1 million events and 15,000 session recordings per month for **free, every month** with **no feature limitations**.
+
+A typical, medium-size B2B product with 10,000 monthly active users, who generate: 
+
+- 100 events each per month
+- 5 sessions each per month
+
+Would use:
+
+- 1 million events per month
+- 50,000 session recordings
+
+Discounts for annual contracts are available on application.
+
+### How much does Mixpanel cost?
+
+Mixpanel prices based on "Monthly Track Users" (MTUs) and applies a limit of 1,000 events per MTU. A company with 10,000 MTUs would pay $190 per month, or £133 per month if you pay annually.
+
+Mixpanel also offers a free tier with a 100k MTU limit, however it applies numerous feature limits – e.g. you can create but **not save** cohorts.
```

Comment:
No CTA at the end of the article?

Reply:
See my earlier comment. CTAs are the last thing I'm going to add, just trying to get the content right atm.